models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md

```diff
@@ -0,0 +1,89 @@
+
+## Example
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Segmentation  |
+| GFLOPs            | 300.801       |
+| MParams           | 4.51          |
+| Source framework  | PyTorch       |
+
+## Accuracy
+
+See []()
+
+The following accuracy metrics are measured on a `brain tumor` training subset of [Medical Decathlon](http://medicaldecathlon.com/) dataset.
+Dice index for "overall": 0.94
+Dice index for "necrotic core / non-enhancing tumor": 0.88
+Dice index for "edema": 0.76
+Dice index for "enhancing tumor": 0.86
+
+Please, note, that the accuracy achieved with ONNX model adapted for OpenVINO&trade can slightly differ from the accuracy achieved with the original PyTorch model since the upsampling operation has been changed from the `trilinear` to `nearest` mode.
+
+## Performance
+
+## Input
+
+The model takes as an input four MRI modalities `T1`, `T1ce`, `T2`, `Flair`. You can find additional information on the BraTS 2019 [page](https://www.med.upenn.edu/cbica/brats2019/registration.html) and [wiki](https://en.wikipedia.org/wiki/Magnetic_resonance_imaging).  
+In the preprocessing pipeline, each modality should be z-score normalized separately. The input tensor is a concatenation of the four input modalities.
+
+### Original model
+
+MR Image, name - `0`, shape - `1,4,128,128,128`, format is `B,C,D,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `D` - depth
+- `H` - height
+- `W` - width
+
+The channels are ordered as `T1`, `T1ce`, `T2`, `Flair`.
+
+### Converted model
+
+MR Image, name - `0`, shape - `1,4,128,128,128`, format is `B,C,D,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `D` - depth
+- `H` - height
+- `W` - width
+
+The channels are ordered as `T1`, `T1ce`, `T2`, `Flair`.
+
+## Output
+
+### Original model
+
+Probabilities of the given voxel to be in the corresponding class, name - `304`, shape - `1,3,128,128,128`, output data format is `B,C,D,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `D` - depth
+- `H` - height
+- `W` - width
+
+With the following channels: `whole tumor`, `tumor core` and `enhancing tumor`.
+
+### Converted model
+
+Probabilities of the given voxel to be in the corresponding class, name - `304`, shape - `1,3,128,128,128`, output data format is `B,C,D,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `D` - depth
+- `H` - height
+- `W` - width
+
+With the following channels: `whole tumor`, `tumor core` and `enhancing tumor`.
+
+## Legal Information
+
+[LICENSE](https://raw.githubusercontent.com/lachinov/brats2019/master/LICENSE)
```

Comment:
Please update the license description according to the new convention (you can use `brain-tumor-segmentation-0001` as an example).